vcd/datasource_vcd_vapp_vm_test.go

```diff
@@ -0,0 +1,87 @@
+// +build vapp ALL functional
+
+package vcd
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+)
+
+// TestAccVcdVappDS tests a vApp data source if a vApp is found in the VDC
+func TestAccVcdVappVmDS(t *testing.T) {
+	// This test requires access to the vCD before filling templates
+	// Thus it won't run in the short test
+	if vcdShortTest {
```

Comment:
shouldn't this check should after template fill for binary tests to work?

Reply:
Just as the comment above says, this cannot happen. 
The short test runs without connection to the vCD, but this test requires connection to the vCD just to determine whether it can run or not.